_episodes/03-create.md

```diff
@@ -87,6 +87,41 @@ nothing to commit (create/copy files and use "git add" to track)
 > ~~~
 > {: .bash}
 >
-> Why is it a bad idea to do this?
+> Why is it a bad idea to do this? (Notice here that the `planets` project is now also tracking the entire `mars` repository.)
 > How can Dracula undo his last `git init`?
+> > ## Solution
+> > Git repositories can interfere with each other if they are "nested" in the
+> > directory of another. We therefore advise to create each new git repository in a
+> > separate directory. Note that we can track files in directories within a git
```

Comment:
I think that "We therefore advise to create each new git repository in a separate directory." is not super clear. What about

> We therefore advise to create new Git repositories **only** when get something like
>
> ~~~
> $ git status
> ~~~
> {: .bash }
> ~~~
> fatal: Not a git repository (or any parent up to mount point /path/to/working/directory)
> Stopping at filesystem boundary (GIT_DISCOVERY_ACROSS_FILESYSTEM not set).
> ~~~
> {: .output }